gateway-messages/src/lib.rs

```diff
@@ -237,6 +237,16 @@ impl<'de> Deserialize<'de> for SpComponent {
     }
 }
 
+impl std::fmt::Display for SpComponent {
```

Comment:
Does this need a `#[cfg(feature = "std")]` guard to not break hubris?

Reply:
Oddly, this didn't _seem_ to break Hubris, but I switched to `core::fmt::Display` for safety (76d0a9879fc6711e5230dc9e0f6f9e95552c5bd3)